cloud-gov/cg-ui
# Replace users list with table

## Changes proposed in this pull request:

- replaces grid list for table in users list
- updates primary color palette settings

### Note

- Per [this decision](https://github.com/cloud-gov/cg-ui/pull/444#issuecomment-2310345988), I went with a horizontal scrollable table over a table with a sticky header

### How to test

1. Log in (or obtain a CF token locally)
2. Navigate to `/orgs/[orgId]`
3. Try out filtering, sorting, both at once
4. Check out design for remove user modal by clicking a "remove" button

Optional: If you want to see last login information in the table, follow these additional steps:

1. Follow [the instructions](https://github.com/cloud-gov/cg-ui?tab=readme-ov-file#s3-user-information) to set up an s3 bucket
2. Obtain a dev CF token locally (`npm run dev-cf`)
5. Set the proper CF targets by running: `cf target -o cloud-gov -s cg-ui`
6. Navigate to `18f-stratos-rebuild` org page: `/orgs/470bd8ff-ed0e-4d11-95c4-cf765202cebd/users/eef1991d-67c8-4650-802b-4c1470bd273a`

### Screenshots

Default:
<img width="1430" alt="Screenshot 2024-08-28 at 2 00 59 PM" src="https://github.com/user-attachments/assets/bc0c00e0-a986-457c-87f5-d6a5a2efe9fd">

Sorted by another column:
<img width="1419" alt="Screenshot 2024-08-28 at 2 01 11 PM" src="https://github.com/user-attachments/assets/da24b5d2-2b67-4531-aed7-765fb3510a24">

Filtered and sorted:
<img width="1430" alt="Screenshot 2024-08-28 at 2 01 21 PM" src="https://github.com/user-attachments/assets/8bc0ac8f-c6b8-42d0-a4be-77c79a8e82e0">

Remove modal:
<img width="1426" alt="Screenshot 2024-08-28 at 2 02 51 PM" src="https://github.com/user-attachments/assets/22d61d95-af42-4347-8a8c-80fc0e687373">

### Related issues

Closes #428 
Closes #414 

### Submitter checklist


## Security considerations

UI only, no new data added
